include/BSMPT/models/SMparam.h

```diff
@@ -228,7 +228,7 @@ struct ISMConstants
 /**
  * @brief imaginary number i
  */
-const std::complex<double> II(0, 1);
+extern const std::complex<double> II;
```

Comment:
Why mark this as extern and move the definition?

Reply:
This is a problem with breathe I believe, I had to change the definition otherwise it would not compile.